README.md

```diff
@@ -1,104 +1,159 @@
-# nucypher-ts
+Full Documentation can be found [here](https://docs.threshold.network/app-development/threshold-access-control-tac).
+# Get Started with TAC
```

Comment:
The new "tutorial section" of README looks very similar to the one in [Threshold docs](https://docs.threshold.network/app-development/threshold-access-control-tac/get-started-with-tac). Is it intended?

Reply:
yes, I wanted to include the basic tutorial from the gitbook because it's a good introduction. The links throughout the readme then link to more detailed information from the gitbook.